test/support/models.rb

```diff
@@ -215,7 +215,7 @@ class ValidatingUser < User
     only_integer: true
   validates_length_of :name, maximum: 25
   validates_length_of :description, maximum: 50
-  validates_length_of :action, maximum: 10, tokenizer: lambda { |str| str.scan(/\w+/) }
+  validates_length_of :words_in_action, maximum: 10
```

Comment:
@rafaelfranca 
I applied your suggestion but still it displays this `deprecation` information. But see how it looks, that's what you suggested right?

```ruby
DEPRECATION WARNING: The `:tokenizer` option is deprecated, and will be removed in Rails 5.1.
You can achieve the same functionality by defining an instance method
with the value that you want to validate the length of. For example,

    validates_length_of :essay, minimum: 100,
      tokenizer: ->(str) { str.scan(/w+/) }

should be written as

    validates_length_of :words_in_essay, minimum: 100

    private

    def words_in_essay
      essay.scan(/w+/)
    end
 (called from <class:ValidatingUser> at /Users/herminiotorres/src/simple_form/test/support/models.rb:218)
```